_translations/de.yml

```diff
@@ -182,6 +182,7 @@ de:
     walletelectrum: "Electrum ist auf Geschwindigkeit und Einfachheit fokussiert und benötigt wenig Ressourcen. Es nutzt entfernte Server, die den komplizierten Teil des Bitcoin-Systems übernehmen und ermöglicht es, mit einer geheimen Phrase wieder Zugriff auf eine Wallet zu bekommen."
     walletmsigna: "mSIGNA ist eine fortgeschrittene aber dennoch leicht zu benutzende Wallet, die Geschwindigkeit, Einfachheit, Skalierbarkeit auf Unternehmensebene und ein hohes Maß an Sicherheit bietet. Es wird BIP32, Multisignatur-Transaktionen, Offline-Speicherung, Synchronisierung mit mehreren Geräten sowie verschlüsselte elektronische und physische Backups geboten."
     walletbitcoinwallet: "Bitcoin Wallet ist einfach zu verwenden und zuverlässig, gleichzeitig aber sicher und schnell. Der Fokus ist auf Dezentralisierung und Zero Trust gerichtet: Kein zentraler Dienst wird für Bitcoin-bezogene Aktionen benötigt. Die App ist eine gute Wahl für technisch nicht versierte Personen."
+    walletedgewallet: "Edge ist eine Segwit-kompatible, mobile Bitcoin-Wallet, die Privatsphäre, Sicherheit und Dezentralisierung für jeden zugänglich und nutzbar macht. Edge-Wallets sind immer automatisch verschlüsselt, gesichert und funktionieren auch dann noch, wenn die Edge-Server nicht erreichbar sind."
```

Comment:
Please revise or drop change to this file as per comment in: `_translations/en.yml`